src/Interfaces/Options/Particles/ISizeAnimation.ts

```diff
@@ -0,0 +1,12 @@
+export interface ISizeAnimation {
+    enable: boolean;
+
+    /**
+     * @deprecated use the new minimumValue instead
+     */
+    size_min: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'size_min' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)